### DISCUSSION TITLE
Notice: Undefined variable: userrow

### DISCUSSION THREAD

**Discussion by Md-Shiblee-Mahbub** [HashenUdara/edoc-doctor-appointment-system#33] [Q&A]
Hello,
Can you solve my problem please, i need to submit this tomorrow. it says...
Notice: Undefined variable: userrow in C:\Program Files\XAMPP\htdocs\edoc-doctor-appointment-system-main\patient\schedule.php on line 47

Fatal error: Uncaught Error: Call to a member function fetch_assoc() on null in C:\Program Files\XAMPP\htdocs\edoc-doctor-appointment-system-main\patient\schedule.php:47 Stack trace: #0 {main} thrown in C:\Program Files\XAMPP\htdocs\edoc-doctor-appointment-system-main\patient\schedule.php on line 47

**Comment by Md-Shiblee-Mahbub** [comment]
After some changes, it's working now.

**Comment by arko19882024** [comment]
@Md-Shiblee-Mahbub , facing same error. What did you modify? Can you please help.

**Reply by Md-Shiblee-Mahbub** [reply]
ya sure, i'll upload the file if you need. contact me.

**Comment by komals109** [comment]
@Md-Shiblee-Mahbub 
Facing the same problem ,can you help? pls
